src/features/rewards/modalActivity/style.ts

```diff
@@ -79,9 +95,22 @@ export const StyledWarning = styled<{}, 'div'>('div')`
   display: flex;
   justify-content: center;
   border-top: 1px solid #ebecf0;
+  border-bottom: 1px solid #ebecf0;
   margin: 0 -50px;
-  padding: 17px 50px 0;
+  padding: 17px 50px 17px;
   align-items: flex-start;
+  white-space: normal;
+`
+
+export const StyledTable = styled<{}, 'div'>('div')`
+  margin: -24px 0;
```

Comment:
This will cause an overlap if the default margin changes of the Table component changes. Can we simply call something like?
```
styled(Table)`
  margin: 0;
`
```

The whole situation isn't ideal, but somehow resetting it to `0` is better than negative-offsetting the currently-set margin value.